temporal-admin-tools/justfile

```diff
@@ -0,0 +1,66 @@
+set quiet # Recipes are silent by default
+set export # Just variables are exported to environment variables
+
+[private]
+default:
+	just --list
+
+[private]
+start-local-registry: (stop-local-registry)
+	docker run -d -p 5000:5000 --name registry registry:2.7
+
+[private]
+stop-local-registry: 
+	docker stop registry || true
+	docker rm registry || true
+
+[private]
+push-to-local-registry version:
+	#!/usr/bin/bash
+	set -euox pipefail
+
+	rock_file=$(ls "${version}" | grep "\.rock")
+	version="${version#temporal-server/}"
+
+	rockcraft.skopeo --insecure-policy copy --dest-tls-verify=false \
+		"oci-archive:${version}/${rock_file}" \
+		"docker://localhost:5000/temporal-admin-tools-dev:${version}"
+
+# Clean the rockcraft project of the component and remove any existing rock files
+clean version:
+	cd "${version#temporal-admin-tools/}" && rockcraft clean
+	cd "${version#temporal-admin-tools/}" && rm -f "$(ls | grep *.rock)"
```

Comment:
I wonder if all of these can be abstracted to a higher level justfile, or some sort of common justfile. It'll be reused in all our rocks' directories. Not a blocker if not possible.

Reply:
The abstraction was introduced in the temporal-server tests PR. There is a hierarchy of justfiles designed in a manner that allows one to run targets from any directory